components/google_cloud/actions/switch-instance-boot-status/switch-instance-boot-status.mjs

```diff
@@ -0,0 +1,63 @@
+import googleCloud from "../../google_cloud.app.mjs";
+
+export default {
+  name: "Switch Instance Boot Status",
+  version: "0.0.1",
+  key: "google_cloud-switch-instance-boot-status",
+  type: "action",
+  description: "Switch a virtual machine instance boot status to start or stop it. [See the documentation](https://cloud.google.com/compute/docs/instances/stop-start-instance)",
```

Comment:
```suggestion
  description: "Switch a virtual machine instance boot status to start or stop it. [See the documentation](https://cloud.google.com/compute/docs/instances/stop-start-instance)"
```

Reply:
Changed 'Switch a virtual machine instance boot status to start or stop it.' to 'Switch a virtual machine instance boot status to start or stop it.' (Google's Developer docs style guide: Use present tense)